mne/bem.py

```diff
@@ -980,7 +980,7 @@ def _check_origin(origin, info, coord_frame='head', disp=False):
 @verbose
 def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
                        volume='T1', atlas=False, gcaatlas=False, preflood=None,
-                       show=False, verbose=None):
+                       show=False, verbose=None, force_copy=False):
```

Comment:
This was the for original PR.. anyone else in favor of one or the other
alternative ?

On Saturday, 21 May 2016, Alexandre Gramfort <notifications@github.com>
wrote:

> In mne/bem.py
> <https://github.com/mne-tools/mne-python/pull/3241#discussion_r64130144>:
>
> > @@ -980,7 +980,7 @@ def _check_origin(origin, info, coord_frame='head', disp=False):
> >  @verbose
> >  def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
> >                         volume='T1', atlas=False, gcaatlas=False, preflood=None,
> > -                       show=False, verbose=None):
> > +                       show=False, verbose=None, force_copy=False):
>
> rather than making the API more complex you should try to detect the pb
> and switch to a copy with a warning message.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3241/files/e22b20312f6851c905de99f65be8b494a3946ac3#r64130144>
>